File: app/src/main/java/com/passionpenguin/ditiezu/HtmlTextView/WrapperTagHandler.java
package com.passionpenguin.ditiezu.HtmlTextView;

import android.text.Editable;

import androidx.annotation.Nullable;

import org.xml.sax.Attributes;

public interface WrapperTagHandler {
    boolean handleTag(boolean opening, String tag, Editable output, @Nullable Attributes attributes);
}

File: app/src/main/java/com/passionpenguin/htmltextview/WrapperTagHandler.kt
package com.passionpenguin.htmltextview

import android.text.Editable
import org.xml.sax.Attributes

interface WrapperTagHandler {
    fun handleTag(
        opening: Boolean,
        tag: String?,
        output: Editable?,
        attributes: Attributes?
    ): Boolean
}